examples/scaling-criteo/03-Training-with-Merlin-Models.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>I'd remove this if this is only for the validation_callback param in the model.fit(). you do not need this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>